ote/src/clj/ote/services/taxiui_service.sql

```diff
@@ -23,4 +23,18 @@ SELECT id,
  LIMIT 1;
 
 -- name: fetch-services-with-prices
-SELECT DISTINCT service_id FROM taxi_service_prices;
\ No newline at end of file
+SELECT DISTINCT service_id FROM taxi_service_prices;
+
+-- name: list-pricing-statistics
+SELECT o.id AS "operator-id",
+       s.id AS "service-id",
+       CONCAT(o.name, '/', s.name) AS name,
+       l.timestamp AS timestamp,
+       l.start_price_daytime AS "start-price-daytime",
+       l.start_price_nighttime AS "start-price-nighttime",
+       l.start_price_weekend AS "start-price-weekend",
+       l.price_per_minute AS "price-per-minute",
+       l.price_per_kilometer AS "price-per-kilometer"
+  FROM list_taxi_pricing_statistics(:column, :direction) l
+  JOIN "transport-service" s ON l.service_id = s.id
+  JOIN "transport-operator" o ON s."transport-operator-id" = o.id;
```

Comment:
For consistency it could be better to use quotes in all column names.

Reply:
The autoformatter seemed to quote only ambiguous columns, so yeah, I'll fix that.